telethon/telegram_bare_client.py

```diff
@@ -9,6 +9,7 @@
 from threading import Lock
 from time import sleep
 
+import telethon
```

Comment:
Oh no, absolute imports. The whole library tries to avoid them for a reason (so you can renam the package and it will still work). There's no way to avoid this?

Reply:
We can put `__version__` into `telethon.version`, and then in `telethon.__init__`:
```py
from . import version
__version__ = version.__version__
```
as setuptools does it.

Reply:
@Lonami Fixed.